Telemetry/CorePing.swift

```diff
@@ -10,14 +10,18 @@ private let PrefKeyProfileDate = "PrefKeyProfileDate"
 private let PrefKeyPingCount = "PrefKeyPingCount"
 private let PrefKeyClientID = "PrefKeyClientID"
 private let PrefKeyModel = "PrefKeyModel"
+private let PrefKeyUsage = "PrefKeyUsage"
 
 // See https://gecko.readthedocs.org/en/latest/toolkit/components/telemetry/telemetry/core-ping.html
-private let PingVersion = 3
+private let PingVersion = 4
```

Comment:
It's a little unfortunate these version numbers aren't aligning with the android implementation but perhaps that's to be expected.